src/graph/backend/dnnl/patterns/utils.hpp

```diff
@@ -301,15 +301,14 @@ inline bool is_int8_quantization(const op_t *op) {
 // 2nd input of BiasAdd.
 inline graph::utils::pm::repetition_t *optional_bias_add(
         const std::shared_ptr<graph::utils::pm::pb_graph_t> &pgraph,
-        graph::utils::pm::pb_op_t *input, bool maybe_typecast = false) {
+        graph::utils::pm::pb_op_t *input, bool maybe_typecast = false,
+        bool check_f32_input = false) {
```

Comment:
It seems this check is not used?

Reply:
rm now. thanks

Reply:
make test
set test_scope=NIGHTLY
disable benchdnn_all
enable benchdnn_graph